app/main/constants/config-const.js

```diff
@@ -6,7 +6,7 @@ angular.module('main')
   ENV: {
     /*inject-env*/
     'SERVER_URL': 'http://sample-env.qtbvngb2iz.us-west-2.elasticbeanstalk.com/',
```

Comment:
do you want to name the server in AWS to something readable? like prod-env instead of sample-env